core/src/main/resources/templates/error.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head><title>Request handling failed</title></head>
+	<body th:with="httpStatus=${T(org.springframework.http.HttpStatus).valueOf(#response.status)}">
+        <h1 th:text="|${httpStatus} - ${httpStatus.reasonPhrase}|">404</h1>
```

Comment:
Agreed. As testing with 404 is the most easy one, I changed it to "406 Not Acceptable", so it's clear that the substitution works when testing for 404.